src/components/NavBar/NavBar.jsx

```diff
@@ -0,0 +1,209 @@
+// React Imports
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+// Solid Imports
+import { useSession, LoginButton } from '@inrupt/solid-ui-react';
+// Material UI Imports
+import AccountCircle from '@mui/icons-material/AccountCircle';
+import AppBar from '@mui/material/AppBar';
+import Badge from '@mui/material/Badge';
+import Box from '@mui/material/Box';
+import Button from '@mui/material/Button';
+import IconButton from '@mui/material/IconButton';
+import Menu from '@mui/material/Menu';
+import MenuItem from '@mui/material/MenuItem';
+import MenuIcon from '@mui/icons-material/Menu';
+import NotificationsIcon from '@mui/icons-material/Notifications';
+import Tab from '@mui/material/Tab';
+import Tabs from '@mui/material/Tabs';
+import Toolbar from '@mui/material/Toolbar';
+import Typography from '@mui/material/Typography';
+// Custom Component Imports
+import LogoutModal from '../LogoutModal/LogoutModal';
+import NavMenu from './NavMenu';
+import NavMenuMobile from './NavMenuMobile';
+import { SOLID_IDENTITY_PROVIDER } from '../../utils';
```

Comment:
Ordering all imports in the following order:

- React
- Inrupt
- Styling/MUI
- Utils
- Hooks
- Contexts
- Constants
- Components